README.md

```diff
@@ -22,6 +22,9 @@ npm install hpt-validator
 npm run test
 ```
 
+#### Node Observations
+If you are running the following command, it has been observed that Node versions 20+ encounter typescript errors. The test suite will run using Node version 19.*.*.
```

Comment:
This is talking about the command on line 22, right? So I think it would be the "previous" command, not the "following" command.